test/suites/integration/scheduling_test.go

```diff
@@ -270,10 +270,12 @@ var _ = Describe("Scheduling", Ordered, ContinueOnFailure, func() {
 		test.ReplaceRequirements(nodePool,
```

Comment:
Going to add scheduling tests for _all_ of the label domain exceptions rather than just this single one